README.md

```diff
@@ -16,7 +16,7 @@ Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/
 
 - Timezone, listens for messages that contain 1:30pm UTC and replies with common timezones translation
 
-- Code of Conduct
+- Flags violations of the Code of Conduct.
```

Comment:
```suggestion
- Flags violations of the Code of Conduct
```
Think about the pint at the end. The rest of the list is without points at the end.